ext/opencensus_trace.c

```diff
@@ -250,7 +252,7 @@ static opencensus_trace_span_t *opencensus_trace_begin(zend_string *function_nam
         php_mt_srand(GENERATE_SEED());
     }
 #endif
-    span->span_id = (php_mt_rand() >> 1);
+    span->span_id = ((uint32_t) php_mt_rand()) >> 1;
```

Comment:
Thanks. It might be a dumb question.

The return value of `php_mt_rand` is `uint32_t` (https://github.com/php/php-src/blob/869fccea24ac829de8ebf9947f6515f03b7d1fe1/ext/standard/mt_rand.c#L163) and I still don't understand why it works.